webrender/src/render_backend.rs

```diff
@@ -1111,6 +1125,13 @@ impl RenderBackend {
             self.result_tx.send(msg).unwrap();
             profile_counters.reset();
         } else if requested_frame {
+
+            // If frame is not rebuilt since last frame rendering and composite frame is
+            // not requested, there is no need to render frame.
+            if render_frame && !doc.rendered_frame_is_valid {
```

Comment:
I think that the meaning of `rendered_frame_is_valid` is reversed throughout the commit (here we should skip rendering if `rendered_frame_is_valid` is `true`).

Reply:
Oh, I made mistake during update the patch in bugzilla.